pkg/apis/admissionregistration/types.go

```diff
@@ -929,3 +949,26 @@ type ServiceReference struct {
 	// +optional
 	Port int32
 }
+
+// MatchCondition represents a condition which must by fulfilled for a request to be sent to a webhook.
+type MatchCondition struct {
+	// Name is an identifier for this match condition, used for strategic merging of MatchConditions,
+	// as well as providing an identifier for logging purposes. A good name should be descriptive of
+	// the associated expression.
+	// Name must be a valid RFC 1123 DNS subdomain, and unique in a set of MatchConditions.
+	// Required.
+	Name string
+
+	// Documentation on CEL: https://kubernetes.io/docs/reference/using-api/cel/
+	//
```

Comment:
nit: move the CEL reference link to below the rest of the comments (above `// Required.`)

Reply:
moved